src/contracts/interfaces/IAVSDirectory.sol

```diff
@@ -2,58 +2,143 @@
 pragma solidity >=0.5.0;
 
 import "./ISignatureUtils.sol";
+import "./IStrategy.sol";
 
 interface IAVSDirectory is ISignatureUtils {
-    /// @notice Enum representing the status of an operator's registration with an AVS
+    /// @notice Enum representing the registration status of an operator with an AVS.
     enum OperatorAVSRegistrationStatus {
-        UNREGISTERED,       // Operator not registered to AVS
-        REGISTERED          // Operator registered to AVS
+        UNREGISTERED, // Operator is not registered with the AVS.
+        REGISTERED // Operator is registered with the AVS.
+
+    }
+
+    struct OperatorSet {
+        address avs;
+        uint32 id;
+    }
```

Comment:
This struct really isn't being utilized well.

In 99% of places where the actual code deals with operator sets, we aren't using this struct. We only use it in events, and as an additional field in `StandbyParam`

However, even in this latter case, the use of a nested struct in `StandbyParam` makes `updateStandbyParams` really messy.

IMO we should just change `StandbyParam` to have all 3 fields rather than a nested struct.